src/components/table/Table.tsx

```diff
@@ -51,17 +84,87 @@ const Table = <T = any,>({
 
   const loading: SpinProps | boolean | undefined =
     typeof props.loading == 'object'
-      ? {
-          ...props.loading,
-          ...defaultLoadingProps,
-        }
+      ? { ...props.loading, ...defaultLoadingProps }
       : props.loading && defaultLoadingProps
 
   const columns = props.columns?.map((column) => ({
     ...column,
     sortIcon: column.sortIcon ?? defaultSortIcon,
   }))
 
+  const isSmallPagination =
+    props.pagination &&
+    typeof props.pagination === 'object' &&
+    props.pagination.size === 'small'
+
+  const paginationLocale = {
+    jump_to: isSmallPagination ? (
+      <>
+        Go to
+        <span style={{ paddingLeft: '4px', paddingRight: '8px' }}>:</span>
+      </>
+    ) : (
+      'Go to'
+    ),
+    page: isSmallPagination ? '' : 'page',
+  }
+
+  const itemRender = (
+    _page: number,
+    type: 'page' | 'prev' | 'next' | 'jump-prev' | 'jump-next',
+    element: React.ReactNode
+  ) => {
+    if (type === 'prev') {
+      return (
+        <a>
+          {paginationTextLabels
+            ? 'Previous'
+            : renderIconButton(<CaretLeftIcon />, isSmallPagination)}
+        </a>
+      )
+    }
+
+    if (type === 'next') {
+      return (
+        <a>
+          {paginationTextLabels
+            ? 'Next'
+            : renderIconButton(<CaretRightIcon />, isSmallPagination)}
+        </a>
+      )
+    }
+
+    if (type === 'jump-prev') {
+      return renderJumpItem(
+        <DotsThreeIcon />,
+        <CaretDoubleLeftIcon />,
+        isSmallPagination
+      )
+    }
+
+    if (type === 'jump-next') {
+      return renderJumpItem(
+        <DotsThreeIcon />,
+        <CaretDoubleRightIcon />,
+        isSmallPagination
+      )
+    }
+
+    return element
+  }
+
+  const paginationConfig = props.pagination
+    ? {
+        ...props.pagination,
+        locale: { ...paginationLocale, ...props.pagination.locale },
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        selectComponentClass: (selectProps: any) => (
```

Comment:
Are Are these select props the same prop types as the props for an antd select? You might be able to import them, the type I think maybe is `SelectProps` potentially. In general unless absolutely necessary I don't like using `any`